homeassistant/components/iammeter/sensor.py

```diff
@@ -36,7 +38,15 @@
 PLATFORM_TIMEOUT = 8
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+async def async_setup_entry(
+    hass: HomeAssistantType, entry: ConfigEntry, async_add_entities
+):
+    """Load the saved entities."""
+    await async_setup_platform(hass, entry.data, async_add_entities)
+    return True
+
+
+async def async_setup_platform(hass, config, async_add_entities):
     """Platform setup."""
     config_host = config[CONF_HOST]
```

Comment:
Import the legacy config to a config flow here. Move the setup code to `async_setup_entry`.